src/components/Footer/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Hyperlink } from '@edx/paragon';
+
+import './Footer.scss';
+
+
+const LinkList = footerLinks => (
+  <ul>
+    {
+      footerLinks.map(link =>
+        (
+          <li>
+            <Hyperlink
+              className="footer-link"
+              destination={link.destination_link}
+              content={link.label_text}
+            />
+            <div className="link-divider">/</div>
+          </li>
+        ))
+    }
+  </ul>
+);
+
+
+const Footer = props => (
+  <footer className="page-footer">
+    <div className="footer-content">
+      <img className="site-logo" alt="site logo" src={props.siteLogo} />
+      {LinkList(props.footerLinks)}
```

Comment:
Similar to the discussion on #38, these "sub-components" should either be actual components or render methods on a Footer class-style component. 
i.e.
```
const LinkList = props => (
  ...
  props.footerLinks.map(link => ())
  ...
)
const Footer = props => (
  ...
  <LinkList footerLinks=props.footerLinks />
)
```
OR
```
class Footer extends React.component({
  renderLinkList(){
    return(
      this.prop.footerLinks.map()
    )
  }
  render(){
    const linkList = renderLinkList();
    ...
    {linkList}
  }
})
```